optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/swingui/VehicleRoutingPanel.java

```diff
@@ -120,7 +114,9 @@ protected Customer createCustomer(VehicleRoutingSolution solution, Location newL
             TimeWindowedCustomer newTimeWindowedCustomer = new TimeWindowedCustomer();
             TimeWindowedDepot timeWindowedDepot = (TimeWindowedDepot) solution.getDepotList().get(0);
             long windowTime = (timeWindowedDepot.getDueTime() - timeWindowedDepot.getReadyTime()) / 4L;
-            long readyTime = RandomUtils.nextLong(demandRandom, windowTime * 3L);
+            long readyTime = demandRandom.longs(0, windowTime * 3L)
+                    .findAny()
+                    .orElseThrow();
```

Comment:
idem

Reply:
I restored the random impl everywhere else to the original, maintaining status quo. (Since when do we question the correctness of JDK methods?)

This particular use is only for the creation of a new customer in the UI, therefore even if the random sequence changed, it would not matter.

Reply:
Agreed on para 2 - even though that changing the random numbers might affect the behavior when doing live demos (as it might generate different random numbers), and theorically some random numbers can be a pain to demo well (for example multiple locations that are pretty much on the same pixel or almost the same pixel). Less surprises during live demos is good. But as it's still seeded, it's still reproducible, so this is probably 100% fine and if not, it will show up during the demo preperation.

As for para 1 - intresting argument, let's discuss. Thanks for taking it out of this PR.